keylime/crypto.py

```diff
@@ -139,7 +139,7 @@ def strbitxor(a,b):
     retval = bytearray(len(b))
     for i in range(len(a)):
         retval[i] = a[i] ^ b[i]
-    return retval
+    return bytes(retval)
```

Comment:
We need to check this does not break later versions (python 37 & >).

See @jetwhiz comment in gitter:

bytes-like is only supported in 2.5+ of python cryptography (https://github.com/pyca/cryptography/commit/3c68250ad9bfb275c760fcce4c72c78c99b57c34#diff-72412af902819141ea548cebf0335477), while python36-cryptography is running 2.3.  so we need to use bytes specifically on the older systems

Reply:
i've confirmed that this is fine on fedora 30 with python3.7 and cryptography 2.6.1